Develop (merge ours branches)

kreeper55 opened the pull request:
delete pane (have one bug) need correcting
if you choose more 1 pane, then the algorithm does not sort the panes correctly
i'm sorry for .xml files )  ignore them

kreeper55 committed:
what?

kreeper55 committed:
i update the version

kreeper55 committed:
secondary and middle clicks: not removed selection

kreeper55 committed:
nothing changed

kreeper55 committed:
update gitignore

kreeper55 committed:
add deleteClick. it removes pane, without sort

kreeper55 committed:
in the MenuItem function added deleteClicks
there are bugs, you need adjustments

ForeshadowRU reviewed (changes requested):
Слишком много мусора

kreeper55 committed:
removing corrected track (tracks) via edit -> delete

kreeper55 committed:
added shortcut menu for panels
added the "delete" item to the context menu. When the PCM is on the panel, it is deleted

kreeper55 committed:
fixed code (ContectMenu: delete)
